examples/post_training_quantization/openvino/yolov8/requirements.txt

```diff
@@ -0,0 +1,4 @@
+ultralytics==8.0.5
+onnx
+openvino-dev
+nncf
```

Comment:
NNCF is prerequisite for examples, is not it?

https://github.com/openvinotoolkit/nncf/blob/1223ee0c4075cb01d03f77f02f021bb08fd0c3e5/examples/post_training_quantization/openvino/yolov8/README.md?plain=1#L11

Reply:
I would remove this text rather than nncf from the requirements.txt. From the user standpoint, it should be much more friendly to run only "pip install -r requirements.txt" rather then navigate to another document and read how to install NNCF.

Reply:
The example is part of the NNCF repository and is valid for current commit. nncf in the requirements.txt can be lead to use nncf version from the pypi instead of repository version. I think the dual behavior of setup of example's envariment It's not good practices.

Reply:
Changes according to agreements